sdk/environments/application_ids.go

```diff
@@ -0,0 +1,115 @@
+	"GraphConnectorService":             graphConnectorServiceAppId,
+	"InformationProtectionSyncService":  informationProtectionSyncServiceAppId,
+	"InTune":                            inTuneAppId,
+	"KustoService":                      kustoServiceAppId,
+	"KustoServiceMFA":                   kustoServiceMFAAppId,
+	"LogAnalytics":                      logAnalyticsAppId,
+	"ManagedHSM":                        managedHSMAppId,
+	"MileIqAdminCenter":                 mileIqAdminCenterAppId,
+	"MileIqDashboard":                   mileIqDashboardAppId,
+	"MileIqRestService":                 mileIqRestServiceAppId,
+	"MixedReality":                      mixedRealityAppId,
+	"MicrosoftAzureCli":                 azureCliAppId,
+	"Microsoft365DataAtRestEncryption":  microsoft365DataAtRestEncryptionAppId,
+	"MicrosoftGraph":                    microsoftGraphAppId,
+	"MicrosoftInvoicing":                microsoftInvoicingAppId,
+	"Office365Connectors":               office365ConnectorsAppId,
+	"Office365Demeter":                  office365DemeterAppId,
+	"Office365DwEngineV2":               office365DwEngineV2AppId,
+	"Office365ExchangeOnline":           office365ExchangeOnlineAppId,
+	"Office365ExchangeOnlineProtection": office365ExchangeOnlineProtectionAppId,
+	"Office365InformationProtection":    office365InformationProtectionAppId,
+	"Office365Management":               office365ManagementAppId,
+	"Office365Portal":                   office365PortalAppId,
+	"Office365SharePointOnline":         office365SharePointOnlineAppId,
+	"Office365Zoom":                     office365ZoomAppId,
+	"OneNote":                           oneNoteAppId,
+	"OneProfileService":                 oneProfileServiceAppId,
+	"OssRdbms":                          ossRDBMSAppId,
+	"PeopleCardsService":                peopleCardsServiceAppId,
+	"PolicyAdministrationService":       policyAdministrationServiceAppId,
+	"PowerAppsRuntimeService":           powerAppsRuntimeServiceAppId,
+	"PowerBiService":                    powerBiServiceAppId,
+	"Purview":                           purviewAppId,
+	"RightsManagementServices":          rightsManagementServiceAppId,
+	"ServiceTrust":                      serviceTrustAppId,
+	"Signup":                            signupAppId,
+	"SkypeForBusinessOnline":            skypeForBusinessOnlineAppId,
+	"SpeechRecognition":                 speechRecognitionAppId,
+	"Synapse":                           synapseAppId,
+	"TargetedMessagingService":          targetedMessagingServiceAppId,
+	"TeamsServices":                     teamsServiceAppId,
+	"ThreatProtection":                  threatProtectionAppId,
+	"TrafficManager":                    trafficManagerAppId,
+	"UniversalPrint":                    universalPrintAppId,
+	"WindowsDefenderAtp":                windowsDefenderAtpAppId,
+	"WindowsVirtualDesktop":             windowsVirtualDesktopAppId,
+	"Yammer":                            yammerAppId,
+
+	// TODO: do we still need these?
```

Comment:
I'd suggest yes, at least until they are fully EOL'd everywhere?

```suggestion
	// Deprecated, remove these when APIs reach final EOL
```